drivers/eeprom/Kconfig

```diff
@@ -41,6 +41,15 @@ config EEPROM_AT2X
 	  Enable support for Atmel AT2x (and compatible) I2C/SPI
 	  EEPROMs.
 
+config EMUL_EEPROM_AT2X
```

Comment:
This breaks the existing naming scheme for EEPROM driver Kconfig options.

Reply:
See commend on the CMakeLists.txt for resolution.